docs/contributing/README.md

```diff
@@ -1,9 +1,142 @@
-# Welcome to the Knative community
+# About Knative
 
-Knative is an open source project that anyone in the community can use, improve, and enjoy.
-We'd love you to join us!
+Knative is an open source project with an active development community.
+The project was started by Google but has contributions from a growing number of industry-leading companies. With a clear project scope, lightweight governance model, and clean lines of
+separation between pluggable components, the Knative project establishes an
+efficient contributor workflow.
 
-In this section:
+Knative is a diverse, open, and inclusive community.
 
-- [Contribute to Knative](contributing.md): helpful tips for how to get started contributing to Knative.
-- [About the Knative community](about.md): information about the Knative community and how it's run.
+Knative is designed for different personas:
+
+![Diagram that displays different Audiences for Knative](images/knative-audience.svg)
+
+Knative components offer developers Kubernetes-native APIs for deploying
+serverless-style functions, applications, and containers to an auto-scaling
+runtime.
```

Comment:
It might be good to note eventing components here as well. Maybe something like this?
```suggestion
runtime. Knative applications work equally well with both HTTP and CloudEvents workloads, with a robust catalog of project and community provided event sources.
```

Reply:
This content feels like it should move up to the main / marketing page, rather than moving down into "contributing" -- I did copy over the front & center of the different audiences, but it seems like our public answer for "what is Knative and what is it for" should align with our community / contributor answer to the same.

That means I didn't merge in the "what Knative components offer" here, nor the "Knative components are intended to be integrated into ..." (I'm not sure I agree with the latter take, though I think Knative is a good component to integrate into those platforms).